catalogue/webapp/components/SearchPageLayout/SearchPageLayout.tsx

```diff
@@ -264,13 +264,17 @@ const SearchLayout: FunctionComponent<{
   );
 };
 
-export const getSearchLayout = (page: ReactElement): JSX.Element => (
-  <SearchLayout
-    hasEventsExhibitions={page.props.serverData.toggles.hasEventsExhibitions}
-    apiToolbarLinks={page.props.apiToolbarLinks}
-  >
-    {page}
-  </SearchLayout>
-);
+export const getSearchLayout = (page: ReactElement): JSX.Element => {
+  const { searchPageEventsExhibitions } = page.props.serverData.toggles;
```

Comment:
we have a [function for grabbing a toggle](https://github.com/wellcomecollection/wellcomecollection.org/blob/28920cf8f63b9995adde8c3d3e55cf94b47e068f/catalogue/webapp/pages/works/%5BworkId%5D/items.tsx#L120), if you want to use it.